packages/algob/src/internal/util/scripts-runner.ts

```diff
@@ -18,17 +18,32 @@ async function loadScript (relativeScriptPath: string): Promise<any> {
   }
 }
 
+// returns error line number with path. eg: scripts/2-gold-asc.js => scripts/2-gold-asc.js:11:3
```

Comment:
Please add doc string for passing variables

Reply:
ok